src/js/Component/Button.js

```diff
@@ -1,3 +1,5 @@
+(function($,Wui) {
```

Comment:
This wrapper needs to be removed on every file it is included on before merging into the master for the following reasons:

1. Its a piece of repeated code that is best included/added by the build script.
2. If seeking to test object in an uncompiled form, that testing should be done in an environment where just that element is being tested, or just directly related components so that the test is simplified enough to be easily reproducible.
3. This wrapper is for a production/integrated environment, not testing, and that is why it should be added by the build script.
4. As we progress to adding a require.js type wrapper on the testing files, we'll have to remove all of this code before that can be added.